pom.xml

```diff
@@ -55,7 +55,7 @@
     </scm>
 
     <properties>
-        <version.elytron>1.1.0.Beta28</version.elytron>
+        <version.elytron>1.1.0.Beta33</version.elytron>
```

Comment:
We need to keep changes like this isolated, now I am in the middle of tagging this will cause a conflict as I upgrade to 1.1.0.Beta34